ibtk/include/ibtk/libmesh_utilities.h

```diff
@@ -1032,7 +1032,6 @@ get_max_edge_length(const libMesh::Elem* const elem, const MultiArray& X_node)
 {
 #ifndef NDEBUG
     TBOX_ASSERT(elem->n_nodes() == X_node.shape()[0]);
-    TBOX_ASSERT(elem->dim() == X_node.shape()[1]);
```

Comment:
@drwells can you take a look at this? Is there a more appropriate assertion to include here?